redis/src/cluster.rs

```diff
@@ -834,16 +794,19 @@ fn get_slots(connection: &mut Connection, tls_mode: Option<TlsMode>) -> RedisRes
     Ok(result)
 }
 
-fn build_connection_string(host: &str, port: Option<u16>, tls_mode: Option<TlsMode>) -> String {
-    let host_port = match port {
-        Some(port) => format!("{}:{}", host, port),
-        None => host.to_string(),
-    };
+fn get_connection_addr(node: (String, u16), tls_mode: Option<TlsMode>) -> ConnectionAddr {
```

Comment:
If this commit is about `TlsMode`, don't change the name and signature for this function as part of this commit.